### ISSUE TITLE
Suggestion: Better selection of font colors in challenges

### ISSUE THREAD

**Issue by nickpapoutsis** [FreeCodeCamp/FreeCodeCamp#5719]
Accurate color representation on a computer screen is difficult and most screens without a proper calibration will not depict the colors accurately enough.

Additionally, there are millions of people with some kind of color blindness which would not be able to identify certain colors (or color changes).

On Waypoint "Target the Children of an Element Using jQuery" we are being asked to change the color of some (rather small) text from the default black to (css) green.

Depending on the angle and distance you are sitting from the screen as well as the screen itself, distinguishing the (css) green from the black might not an easy task (the same is also true for other color combinations).

![capture green](https://cloud.githubusercontent.com/assets/13612952/12071945/ae8b0536-b0cf-11e5-9063-84377eda54f9.PNG)
![capture black](https://cloud.githubusercontent.com/assets/13612952/12071946/ae8e23ec-b0cf-11e5-80f0-4e35daf1963b.PNG)

(to observe the "phenomenon" click on the screenshots to view them in full size)

I suggest that for all such challenges a better color combination is chosen.

**Comment by SaintPeter** [comment]
I agree, this can be hard to see on some screens, it's pretty dark.  Maybe a different color like purple?  I don't know - something higher contrast?

**Comment by nickpapoutsis** [comment]
Darker colors such as (css) Green, Purple or Brown are generally harder to distinguish so I would suggest either Orange or Pink because both are easily readable and well know as colors.

Although we could get a bit more creative (while introducing campers to the plethora of css color names) and use colors such as Aqua, Cyan, Fuchsia, Lime, Magenta, Violet or Turquoise.
